wordpress-cloud-rds/dev/platform.k

```diff
@@ -3,12 +3,13 @@ import catalog.models.schema.v1 as ac
 # platform.k declares customized configurations 
 {{ .AppName }}: ac.AppConfiguration {
     database: {
-        category = "serverless_basic"
+        privateRouting = False
+
         # SubnetID defines the virtual subnet ID associated with the VPC that the rds
         # instance will be created in. The rds instance won't be created in user's own VPC
-        # if this field is not provided.
-        subnetID = [your-subnet-id]
-        securityIPs = ["0.0.0.0/0"]
-        privateRouting = False
+        # but in default VPC of cloud vendor, if this field is not provided.
+        # subnetID = [your-subnet-id]
```

Comment:
Uncomment this field as it is necessary. Users cannot succeed without it.

Reply:
The cloud rds in `dev/main.k` has defaulted to `AWS` RDS instance, and the attributes `subnetID` & `category` is not needed when provisioning an `AWS` RDS. If users want to use `Alicloud`, the related documentation has been updated in this [PR](https://github.com/KusionStack/kusionstack.io/pull/323) of kusionstack.io.